### ISSUE TITLE
How to make a boosted phrase with FluentAPI?

### ISSUE THREAD

**Issue by stas-lazarev-tga** [Shazwazza/Examine#386]
Hello @Shazwazza 

First of all thank you!
Great work on this package👍 

---

As of  `v3.2.0`,
It seems that the FluentAPI is **missing an option to apply boosting on a phrase**. Or at least I can't find how.

We can make a phrase with `.Escape()` like this:
```cs
externalIndex.Searcher.CreateQuery("content")
	.Field( "Title", "some phrase".Escape() );
```

And we can make a boosted term with `.Boost()` like this:
```cs
externalIndex.Searcher.CreateQuery("content")
	.Field( "Title", "potato".Boost(10) );
```

But how do I apply both?
How do I make it generate the following LuceneQuery using your FluentAPI methods:
```
Title:"some phrase"^10
```

---

FYI, In order to check the LuceneQuery string that is generated by FluentAPI methods I'm just calling a regular `.ToString()` on a query. Like this:
```cs
var queryString = externalIndex.Searcher.CreateQuery("content")
	.Field("Title", "some phrase".Escape())
	.ToString();
```

**Comment by Shazwazza** [comment]
Yes Examine API currently doesn't support chaining IExamineValue like .Escape().Boost(10)

A work around is to use a NativeQuery like 

.NativeQuery("+phrase:\"hello world\"^10")

I'll look into adding this behavior but will most likely not be with chaining but the ability to pass in a boost to methods like Escape.

**Comment by Shazwazza** [comment]
This is a related topic: https://github.com/Shazwazza/Examine/issues/329#issuecomment-2168484838

**Comment by Shazwazza** [comment]
This now works in v4, see this syntax: https://github.com/Shazwazza/Examine/blob/release/4.0/src/Examine.Test/Examine.Lucene/Search/FluentApiTests.cs#L213

There is a new WithBoost method that can be applied to any `Examiness`
